src/main/java/com/feelfreecincinncati/FeelFreeCincinnatiController.java

```diff
@@ -18,7 +18,7 @@
 	@Autowired
 	private iAdventureService adventureServiceStub;
 	@Autowired
-	private EventServiceStub eventServiceStub; 
+	private iEventService eventServiceStub; 

```

Comment:
Types in Java are UpperCamelCase, so make this IEventService instead of iEventService.  Also, change the variable name, as it currently indicates that the object will be a stub, but with an interface as a variable type, the object type may not be a stub.